contents/blog/what-is-a-growth-engineer.md

```diff
@@ -0,0 +1,108 @@
+---
+title: What is a growth engineer (and how they help your startup)
+date: 2023-04-19
+author:
+    - ian-vanagas
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: Engineering
+tags:
+ - Explainers
+---
+
+Every startup loves to see signups, subscriptions, and revenue go up, yet all engineers basically work on things indirectly related to doing so. The power of growth engineers is that they focus on making these numbers go up, working on whatever it takes.
+
+A growth engineer is an engineer who works on projects impacting the core business metrics, no matter where they are in the product. This is different than regular engineers who work on specific products and features to improve metrics for them. This indirectly improves the core business metrics.
```

Comment:
"Growth engineering" roles can be _very_ top-of-funnel focused at some companies. Some growth engineers I've talked to don't even code and they are more part of the marketing team. Might be worth specifying that we're talking about growth engineering roles that put a heavy emphasis on the "engineering" part.

Reply:
Got it, I mentioned this in a few places now.